scripts/deploy/secrets.sh

```diff
@@ -96,5 +86,4 @@ revert() {
     upload_secrets
 }
 
-check_not_empty TASK ENVIRONMENT
```

Comment:
This is covered by `set -u` and lines 5 and 6. i.e. if those args are undefined the script fails.

Reply:
```
$ .make secrets-download
./scripts/deploy/secrets.sh: line 6: $2: unbound variable
make: *** [secrets-download] Error 1
```

You're technically correct, but the error is not very useful. I would suggest to keep this error handling.

Reply:
> ```
> $ .make secrets-download
> ./scripts/deploy/secrets.sh: line 6: $2: unbound variable
> make: *** [secrets-download] Error 1
> ```
> 
> You're technically correct, but the error is not very useful. I would suggest to keep this error handling.

This is the case with the current handling i.e. `check_not_empty` does not run due to `set -u` https://github.com/stackrox/infra/blob/master/scripts/deploy/secrets.sh

Reply:
Should we remove `-u` then?

Reply:
> Should we remove `-u` then?

`set -u` is a good thing in bash scripts. I'd like to merge this change as the script behaves as before with less code which is a net improvement IMO.